### PR TITLE
Fix the HTML to validate with W3C Markup Validation Service

### PR DESCRIPTION
**Pull request by tomasz1986**

### CODE REVIEW

**Comment by tomasz1986** [comment]
The local code validates 100% (Home, Downloads, Security, Foundation, Donations, and New GUI & REST API). However, in the preview, there is one error, which I am not sure where is coming from.

Local:
```
<h2 id="contact">Contact</h2>
<p>Send us an email at <a href="mailto:foundation@syncthing.org">foundation@syncthing.org</a>. You can use the form at the bottom of the page if you think it's more convenient, it has the same effect.</p>
```

Preview (and the real website)
```
<h2 id="contact">Contact</h2>
<p>Send us an email at <a href="mailto:foundation@syncthing.org"><a href="mailto:foundation@syncthing.org">foundation@syncthing.org</a></a>. You can use the form at the bottom of the page if you think it's more convenient, it has the same effect.</p>
```

For some reason, `<a href="mailto:foundation@syncthing.org">` is repeated twice. Why is it so? This seems to happen after having been processed by Netlify.

The Foundation and Donations pages fail to validate because of this.

https://validator.w3.org/nu/?showsource=yes&doc=https%3A%2F%2Fdeploy-preview-7--syncthing.netlify.com%2Ffoundation%2F
https://validator.w3.org/nu/?showsource=yes&doc=https%3A%2F%2Fdeploy-preview-7--syncthing.netlify.com%2Fdonations%2F

**Comment by calmh** [comment]
Seems reasonable. I don't understand the `<p>` changes but also don't see anything broken in the preview. I suspect the duplicate mailto link is a Hugo bug. I have 0.62.2 locally, netlify.toml specifies 0.60.1. Could try bumping that and see if it goes away?

**Comment by tomasz1986** [comment]
Thank you for accepting the changes :relaxed:.

The `<p>` modifications are related to the fact that Hugo wrapped the actual text paragraphs with another pair of `<p>` tags. This ended up in overlapping `<p>` tags, which is a no-no. My intention is only to clean up the code, and specifically not to touch the layout, so there should be no differences in the rendering.

To tell the truth, these commits are only aimed to make the code pass the validator, and not necessarily to make it more semantic or similar. Some of the commits are just hacks, e.g. https://github.com/syncthing/website/pull/7/commits/3197d30306d44d2601f286ff8437660d0a334d28 which wraps the image anchor in `<p>`. It should be a `div`, but then we would require additional CSS to add the margins, etc., and I just wanted to make the code valid for now.

To sum up, I work with XML/XHTML on a daily basis, where even one error invalidates the whole file, and I like the keep markup valid at all times in the case of HTML too.

My Hugo version is also 0.62.2. I will experiment with the `netlify.toml` file, so if possible, please wait with merging the commits. I will write back soon.

**Comment by tomasz1986** [comment]
You were spot on right! I have pushed a new commit updating the Netlify configuration to Hugo 0.62.2, and the duplicate elements have disappeared. All pages now validate.

**Comment by calmh** [comment]
Great, thanks for fixing!
